docker-compose.yaml

```diff
@@ -26,29 +27,6 @@ services:
       - "traefik.http.routers.demo.rule=Host(`demo.perses.dev`)"
       - "traefik.http.routers.demo.tls.certresolver=myresolver"
 
-  sandbox:
```

Comment:
Are you sure about removing it? Seems not much people played with this sandbox instance so far, but I think it's good for people new to Perses to be able to try it in edit mode without having to install it by themselves.

Reply:
With permissions, demo instance will not be readonly anymore, so sandbox can be removed